app/javascript/styles/about.scss

```diff
@@ -120,18 +120,55 @@
 }
 
 .information-board {
-  margin: 20px 0;
-  display: flex;
-  justify-content: space-between;
-  border-top: 1px solid lighten($ui-base-color, 10%);
-  border-bottom: 1px solid lighten($ui-base-color, 10%);
-  padding-right: 14px;
+  background: darken($ui-base-color, 4%);
+  padding: 40px 0;
+
+  .panel {
```

Comment:
mostly just a comment, but the heavily nested scss style is much harder to read and maintain then the BEM style we use in the rest of the app. it makes it harder to override the content on the instance-owner side as well.